### ISSUE TITLE
Starting the project is hard for a beginner

### ISSUE THREAD

**Issue by ohmushi** [c-lings/clings#4]
Currently, the project use vcpck as depency manager, and the project requires it.
The beginner should be able to build and run all the project with only one command and find it easily in the README.

I think we should have a script which install the dependency manager if not already installed, install the dependencies, build and run the project.

**Comment by TheRealHoko** [comment]
Yes, a nice detail to take into account in this issue would be to be careful to have an isolated environment for the dependencies. For example a client would have cmake v2.x and right now the latest version of cmake is enforced.

**Comment by ohmushi** [comment]
You're right, we should lower the cmake_minimum_required to the **actulal** non breaking minimum version.

**Comment by Trevodorax** [comment]
Sounds good to me, the script should be fairly fast to do for Unix, but will be trickier to make one that is cross-platform

**Comment by ohmushi** [comment]
There are several problems for a beginner to start with clings :
* s/he have to clone the repo to begin with, and during my first year as a student, i had no clue how to deal git/github
* s/he have to install a C running environment, specifically on windows (at least during my early time as a student)
* because we copy the katas files in the build directory, each modifications implies to re run cmake to replaces those files.

To make things easier, we must create scripts or an other thing that can run stuffs to :
* install if not present cmake / ctest
* install automatically clings, for each OS
* a watch mode that recompile all or at least a command s/he run to re run cmake

The problem with scripts is that they are os dependent. So we have to duplicate (bash/ powershell/ others like nix I guess)
